src/lib.rs

```diff
@@ -91,6 +72,79 @@
 //! #     egui::CentralPanel::default().show(ctx, |ui| my_tabs.ui(ui));
 //! # });
 //! ```
+//!
+//! ## Surfaces
+//!
+//! A [`Surface`] is an abstraction for any tab hierarchy. There are two kinds of
+//! non-empty surfaces: `Main` and `Window`.
+//!
+//! There can only be one `Main` surface. It's the one surface that is rendered inside the
+//! [`Ui`](egui::Ui) you've passed to [`DockArea::show_inside`], or inside the
+//! [`CentralPanel`](egui::CentralPanel) created by [`DockArea::show`].
+//!
+//! On the other hand, there can be multiple `Window` surfaces. Those represent surfaces that were
+//! created by undocking tabs from the `Main` surface, and each of them is rendered inside
+//! a [`Window`](egui::Window) hence their name.
```

Comment:
perhaps clarify here explicitly that it's an ``egui::Window`` in the link, as any reader would need to either hover or click the link in order to fully understand the sentence

Reply:
That's done now, thanks!